src/include/OSL/wide.h

```diff
@@ -310,6 +311,18 @@ template<typename DataT, int WidthT>
 struct Block<DataT*, WidthT> : public BlockOfBuiltin<DataT*, WidthT> {
 };
 
+template <int WidthT>
+struct Block<ClosureColorPtr, WidthT> : public BlockOfBuiltin<ClosureColorPtr, WidthT> {};
+
+template <int WidthT>
+struct Block<ClosureComponentPtr, WidthT> : public BlockOfBuiltin<ClosureComponentPtr, WidthT> {};
+
+template <int WidthT>
+struct Block<ClosureAddPtr, WidthT> : public BlockOfBuiltin<ClosureAddPtr, WidthT> {};
+
+template <int WidthT>
+struct Block<ClosureMulPtr, WidthT> : public BlockOfBuiltin<ClosureMulPtr, WidthT> {};
+
```

Comment:
Try deleting these specializations, as the generic template on lines 310-312 should cover any pointer types.
```
template<typename DataT, int WidthT>
struct Block<DataT*, WidthT> : public BlockOfBuiltin<DataT*, WidthT> {
};
```